packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -227,6 +227,11 @@ export class Viewer {
    */
   @Prop() public experimentalGhostingOpacity = 0;
 
+  /**
+   * Specifies the opacity, between 0 and 1, for a phantom feature, where 0 is completely hidden and 1 is completely visible.
+   */
+  @Prop() public phantomOpacity = 0.5;
```

Comment:
I think the default here should be much lower. @mikemccarty-vertex Did you have any thoughts on what this value should be by default? Maybe 0.3, or even 0.1?

Reply:
Agreed.  It should be pretty _wispy_ by default.  The RCL default is 0.1 which is definitely wispy but (IMO) not outrageously so.

Reply:
This is `opacity=0.1`: 
![image](https://github.com/Vertexvis/vertex-web-sdk/assets/32515649/eba4644d-fb48-4950-aa11-dac33705f07f)